CHANGELOG.md

```diff
@@ -2,12 +2,19 @@
 
 ## dev
 
-Version <dev> of the agent adds [Roda](https://roda.jeremyevans.net/) instrumentation, introduces improved error tracking functionality by associating a transaction id with each error, and uses more reliable network timeout logic.
+Version <dev> of the agent adds [Roda](https://roda.jeremyevans.net/) instrumentation, adds a new `allow_all_headers` configuration parameter to permit capturing all HTTP headers, introduces improved error tracking functionality by associating a transaction id with each error, and uses more reliable network timeout logic.
 
 - **Feature: Add Roda instrumentation**
 
   [Roda](https://roda.jeremyevans.net/) is a now an instrumented framework. The agent currently supports Roda versions 3.19.0+. [PR#2144](https://github.com/newrelic/newrelic-ruby-agent/pull/2144)
 
+- **Feature: New allow_all_headers configuration parameter**
+
+  A new `allow_all_headers` configuration parameter brings parity with the Node.js agent (see the [v2.7.0 changelog](https://docs.newrelic.com/docs/release-notes/agent-release-notes/nodejs-release-notes/node-agent-270/)). This configuration parameter defaults to a value of `false`. When set to `true`, and as long as the agent is not operating in high-security mode, all HTTP headers gleaned from a request will be captured and relayed to New Relic instead of the default core set of headers. All existing behavior for `.*attributes.include` and `.*attributes.exclude` configuration parameters will be respected for any desired filtration of the headers when `allow_all_headers` is enabled. This work was done in response to a feature request submitted by community member [@jamesarosen](https://github.com/jamesarosen). Thank you very much, @jamesarosen! [Issue#1029](https://github.com/newrelic/newrelic-ruby-agent/issues/1029)
+
+  NOTE: The extra headers collected by having `allow_all_headers` enabled requires Rack version 2 or higher.
+  NOTE: The extra headers will appear as attributes prefixed with `request.headers.`
+
```

Comment:
The long block of text makes this changelog entry a bit difficult for me to follow. Would you be open to breaking things into bullets instead? 

```suggestion
  A new `allow_all_headers` configuration option brings parity with the Node.js agent (see the [v2.7.0 changelog](https://docs.newrelic.com/docs/release-notes/agent-release-notes/nodejs-release-notes/node-agent-270/)). 
  
  This configuration option:
    * Defaults to `false`.
    * Is not compatible with high-security mode
    * When set to `true` all HTTP headers gleaned from a request will be captured and relayed to New Relic. By default, only a small group of HTTP headers are sent to New Relic. 
    * Respects all existing behavior for `attributes.include` and `attributes.exclude` configuration options for any desired filtration of the headers when `allow_all_headers` is enabled. 
    *  Requires Rack version 2 or higher.
    * Displays the additional headers as transaction attributes prefixed with `request.headers.`
  
  This work was done in response to a feature request submitted by community member [@jamesarosen](https://github.com/jamesarosen). Thank you very much, @jamesarosen! [Issue#1029](https://github.com/newrelic/newrelic-ruby-agent/issues/1029)
```